packages/smooth_app/lib/pages/hunger_games/congrats.dart

```diff
@@ -66,7 +71,12 @@ class CongratsWidget extends StatelessWidget {
           else
             EMPTY_WIDGET,
           TextButton(
-            child: Text(appLocalizations.close),
+            child: Text(
+              appLocalizations.close,
+              style: TextStyle(
+                color: isDarkMode ? Colors.white : Colors.black,
+              ),
```

Comment:
Same here, but if these colors are indeed needed, please use `Theme.of(context).textTheme.bodyText1!.apply(` as above